forgejo/docker-compose.yml

```
@@ -0,0 +1,44 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: forgejo_server_1
+      APP_PORT: 8101
+      PROXY_AUTH_ADD: "false"
+
+  server:
+    image: codeberg.org/forgejo/forgejo:10.0.1-rootless@sha256:20a12bd39d49554964e059c501d88a021a0ac04c1d3128c24921045013bd16f7
+    user: "1000:1000"
+    restart: on-failure
+    ports:
+      - "${APP_FORGEJO_SSH_PORT}:${APP_FORGEJO_SSH_PORT}"
+    volumes:
+      - ${APP_DATA_DIR}/data/forgejo/data:/var/lib/forgejo
+      - ${APP_DATA_DIR}/data/forgejo/config:/etc/forgejo
```

The data and config directory need to be created in the data/forgejo folder. Also the folder where they are mounted to are named **gitea** in the container.
```suggestion
    volumes:
      - ${APP_DATA_DIR}/data/forgejo/data:/var/lib/gitea
      - ${APP_DATA_DIR}/data/forgejo/config:/etc/gitea
```